res/skins/QMLDemo/Hotcue.qml

```diff
@@ -0,0 +1,52 @@
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+import "Theme"
+
+Item {
```

Comment:
Hotcue doesn't seem to be a visual item so it would be more appropriate to use `
```suggestion
import QtQml 2.12
import "Theme"

Component {
```
Hotcue is not a visual component so this would make more sense

Reply:
Yes, that was my first idea. But this doesn't work, Component does not allow defining js methods. My next idea was to Use Qt Object, but that doesn't allow ControlProxy object in its body.

Reply:
Strange, I misinterpreted about how component should be used then. QtObjects should be the preferable choice then. The only problem is that they have no `default property` so you have to put each one explicitly into a property. doing this for all controlproxies works for me but its also quite ugly. I guess I'm fine with keeping these as an `Item`
```qml
readonly property var hotcueColorControl: Mixxx.ControlProxy {
        id: hotcueColorControl
        group: root.group
        key: "hotcue_" + hotcueNumber + "_color"
    }
```